gelato-relay/tests/relay_integration_test.rs

```diff
@@ -0,0 +1,29 @@
+use gelato_relay::*;
+
+#[tokio::test]
+async fn bindings_query_data() -> Result<(), reqwest::Error> {
+    let gelato = GelatoClient::default();
+
+    let chains = gelato.get_gelato_relay_chains().await.unwrap();
+    println!("Relay chains: {:?}", chains);
```

Comment:
Should these `println` be replaced with `assert`?

Reply:
yep fixed